modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -345,6 +425,17 @@ For information, see the xref:xdcr-reference:xdcr-advanced-settings.adoc#table-o
 
 This setting can be established and retrieved either for an individual replication or globally.
 
+| `conflictLogging`
+| JSON Object
+| Configuration settings for logging conflicts.
+
+This configuration setting defines objects/parameters and options used to control how conflicts are logged during an XDCR replication.
+
+Within the JSON Object, `conflictLogging` setting is enabled by default (`disabled=false`).
+
+This setting uses a JSON object named `loggingRules` to define its values.
```

Comment:
This line, line 436 is out of context and confusing.   Should just remove this line also.